### PR TITLE
ComicExtra doesn't show chapters from multiple pages

### PR DESCRIPTION
**Pull request by Yong-L**
Addresses #1101

### CODE REVIEW

File: 3 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apply plugin: 'com.android.application'
```
Comment (arkon) [inline 3]: Remove this file.

Comment (Yong-L) [reply 3]: Just removed

Comment (arkon) [reply 3]: I mean the whole file, not just the line.

Comment (Yong-L) [reply 3]: Remove the gradle file?

Comment (arkon) [reply 3]: Yes, it's a bad merge conflict resolution.

Comment (Yong-L) [reply 3]: Oh I see. Didn't see that there was created. Just removed it

**Pull request merged**